native/packages/react-native-bpk-component-card/src/__snapshots__/BpkCard-test.android.js.snap

```diff
@@ -8,7 +8,7 @@ exports[`Android BpkCard should render correctly 1`] = `
     accessibilityComponentType="button"
     accessibilityLabel="Example Card"
     background={undefined}
-    onPress={[Function]}
+    onPress={[MockFunction]}
```

Comment:
Just checking, is this. because of the Jest update to 22?

Reply:
Yeah they've changed the serialization of `jest.fn` to be clearer. See https://github.com/facebook/jest/pull/4668